mgmt/obj/ftn.py

```diff
@@ -0,0 +1,116 @@
+
+import logging
+from kubernetes.client.rest import ApiException
+
+logger = logging.getLogger()
+
+class Ftn(object):
+	def __init__(self, obj_api, opr_store, spec=None):
+		self.obj_api = obj_api
+		self.store = opr_store
+		self.droplet_obj = None
+		self.vpc = ""
+		self.vni = ""
+		self.nip = ""
+		self.prefix = ""
+		self.net = ""
+		self.status = OBJ_STATUS.ftn_status_init
+		self.scaled_ep_obj = '/trn_xdp/trn_transit_scaled_endpoint_ebpf_debug.o'
+		if spec is not None:
+			self.set_obj_spec(spec)
+
+	@property
+	def rpc(self):
+		return TrnRpc(self.ip, self.mac)
+
+	def get_obj_spec(self):
+		self.obj = {
+			"name": self.name,
+			"ip": self.ip,
+			"droplet": self.droplet,
+			"mac": self.mac,
+			"chains": self.chains,
+			"nextnode": self.nextnode,
+			"status": self.status

```

Comment:
Have you tried to run these? Where do these variables' values get populated? Please have them in the constructor.